projects/charmfinance/index.js

```diff
@@ -0,0 +1,89 @@
+const sdk = require("@defillama/sdk");
+
+const vaultAbi = require("./vaultAbi.json");
+const cubePoolAbi = require("./cubePoolAbi.json");
+
+const USDT = "0xdac17f958d2ee523a2206206994597c13d831ec7";
+const USDC = "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48";
+const WETH = "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2";
+const WBTC = "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599";
+const ETH = "0x0000000000000000000000000000000000000000";
+
+const VAULT = "0x55535C4C56F6Bf373E06C43E44C0356aaFD0d21A";
+const CUBE_POOL = "0x23F6A2D8d691294c3A1144EeD14F5632e8bc1B67";
+const OPTIONS_CONTRACTS = [
```

Comment:
Any way to get new addresses from github/api/smth?

Reply:
Now it is using directly the yaml file from their github, done by the last commit